documentation/resource_osl_mysql_test.md

```diff
@@ -4,6 +4,10 @@ Installs and initializes a MariaDB service, alongside setting up a user and data
 
 All databases created are encoded as `utf8mb4_unicode_ci` by default.
 
+### CentOS 7 Warning
+
+Due to the age of the version of MariaDB provided in the CentOS 7 distro repo, it is advised to use the `version` property to choose a newer release. We recommend to install version `10.11` if it's on a CentOS 7 system.
```

Comment:
Please state this is only required if you run into problems due to the older version.